core/commons.yaml

```diff
@@ -702,21 +702,21 @@ components:
       - $ref: "#/components/schemas/multilinestringGeoJSON"
       - $ref: "#/components/schemas/polygonGeoJSON"
       - $ref: "#/components/schemas/multipolygonGeoJSON"
-      - $ref: "#/components/schemas/geometrycollectionGeoJSON"
-    geometrycollectionGeoJSON:
-      type: object
-      required:
-        - type
-        - geometries
-      properties:
-        type:
-          type: string
-          enum:
-            - GeometryCollection
-        geometries:
-          type: array
-          items:
-            $ref: "#/components/schemas/geometryGeoJSON"
+    #  - $ref: "#/components/schemas/geometrycollectionGeoJSON"
```

Comment:
This should not be a thing. If we need to do such hacks, I'd not try to combine any longer.